ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -26,11 +33,12 @@ type OwnProps = {
 };
 
 type Props = ReturnType<typeof mapStateToProps> & ReduxProps & OwnProps;
-
 type RequestState = pot.Pot<string, string>;
 
 type State = {
   requestState: RequestState;
+  requestUrl?: string;
+  startingMs: number;
```

Comment:
```suggestion
  startingMillis: number;
```

also consider using `Millisecond` in place of `number`